organization-questionnaire/gsoc2025_proposal.md

```diff
@@ -0,0 +1,144 @@
+participants, both during and after the program, would be a key indicator of
+success. Ultimately, a successful GSoC would not only drive innovation and
+growth for Open Science Labs, but also inspire students to continue contributing
+to open science and open source projects, further increasing the organization's
+impact.
+
+## How will you keep mentors engaged with their GSoC contributors?
+
+At Open Science Labs, the commitment of mentors to their GSoC participants is
+rooted in our core values of collaboration, innovation, and openness, as well as
+our commitment to education and community growth. Mentors at OSL are naturally
+passionate about fostering learning and development, which drives their
+commitment to mentoring students throughout the program. To maintain this
+commitment, we create a supportive environment where mentors feel valued and
+connected to the mission of advancing open science. Regular check-ins,
+collaborative discussions, and shared goals help mentors stay actively involved
+in their contributors' progress. We also emphasize the importance of
+peer-to-peer learning, where mentors not only share their expertise, but also
+gain fresh perspectives from students. By aligning mentorship with our formula
+for success - empowering the community through tools, resources and support - we
+ensure that mentors stay motivated and invested in helping contributors succeed,
+ultimately contributing to the lasting impact of open science and technology.
+
+## How will you help your GSoC contributors stay on schedule to complete their projects?
+
+To help GSoC contributors stay on schedule and complete their projects
+successfully, Open Science Labs would implement a structured yet flexible
+approach. This would include establishing clear milestones and deliverables at
+the outset, scheduling regular check-ins for progress reviews and feedback, and
+encouraging the use of project management tools. Mentors would actively guide
+contributors, fostering open communication and providing support to overcome
+challenges. By breaking down larger goals into manageable tasks and maintaining
+a supportive environment, Open Science Labs would ensure that contributors stay
+motivated, productive, and on track to deliver impactful results. Also, our
+organization has members from multiple countries, and most meetings are held in
+Spanish, Portuguese and English to accommodate everyone. We will invite every
+selected student to these meetings and make them feel welcome.
+
+## How will you keep your GSoC contributors involved in your community during and after GSoC?
+
+To keep GSoC contributors involved in the Open Science Labs community during and
+after GSoC, we will leverage our welcoming and inclusive environment. Our
+community includes members from multiple countries, and meetings are held in
+Spanish, English, and Portuguese to accommodate everyone. We will invite each
```

Comment:
not necessary to mention about language here .. or maybe mention that in a "after gsoc contribution" section .. I mean .. maybe at the end or in the part we mention about after gsoc contributions ... we can mention that osl members are encouraged to organize meetings and activities in their own language to foster the usage and development of the project in their own regions